pkg/minishift/update/update.go

```diff
@@ -282,11 +282,11 @@ func updateBinary(binary string) error {
 		Hash: crypto.SHA256,
 	})
 	if err != nil {
-		fmt.Errorf("Cannot apply binary update: %s", err)
-		err := update.RollbackError(err)
-		if err != nil {
-			return errors.New(fmt.Sprintf("Failed to rollback update: %s", err))
+		rollbackErr := update.RollbackError(err)
+		if rollbackErr != nil {
```

Comment:
> So how does this fix things? Even in the old code RollbackError is called. I have a hard time seeing the difference.

As you said before that `fmt.Errorf` gets ignored and we were not returning it. Now we have err and rollbackErr variable. Here `err` variable keep what `error` returned by `Apply` and rollbackErr keep what returned by `RollbackError`. Previously we were not returning err which came from `Apply` and overwriting it with `RollbackError` but now we do return this err and if rollback fails then Rollback one.